LetsDoStuff.WebApi/appsettings.json

```diff
@@ -1,6 +1,6 @@
 {
   "ConnectionStrings": {
-    "DefaultConnection": "Server=(localdb)\\mssqllocaldb;Database=LetsDoStuffDb;Trusted_Connection=True;"
+    "DefaultConnection": "Server=.\\SQLEXPRESS;Database=LetsDoStuffDb;Trusted_Connection=True;"
```

Comment:
Restore the previous version of this file. We are using a localdb in the app development lifecycle.

Reply:
I've already done this by a commit. It's strange that the old version is still here(